tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -114,7 +114,6 @@ export class Ide {
     async waitNotificationAndOpenLink(notificationText: string, timeout: number) {
         Logger.debug(`Ide.waitNotificationAndOpenLink "${notificationText}"`);
         await this.waitNotification(notificationText, timeout);
-        await this.waitApllicationIsReady(await this.getApplicationUrlFromNotification(notificationText), timeout);
```

Comment:
I'm not about removing this.
Back in the days, the tests sometimes were failing on this issue: https://github.com/eclipse/che/issues/13979. That's why this `waitApllicationIsReady` method was added here - to make sure the application is really running and we can safely open the preview and inspect it.
I'm afraid, that by removing this line we are inviting this flakiness back :-(

Reply:
It was fixed for Eclipse Che 7.1.0
I would propose to remove it right now and then will be observing how it will be going.

Reply:
But on the other hand, this *possible* flakiness can be treated separately.